charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -60,6 +60,22 @@
         }
       }
     },
+    "Service": {
```

Comment:
This is a generated file (from pkg/model). Also, there is already an identical structure in pkg/model/datacatalog.

Reply:
OK. Using datacatalog.ResourceDetails instead. 
I have updated fybrik_module.json manually. Is there a way to generate this file automatically?